### ISSUE TITLE
Suggestion to optimize Glean telemetry collection.

### ISSUE THREAD

**Issue by data-sync-user** [mozilla/uniffi-rs#700]
@chutten pointed out to me that A-S is recording several things as counts along with an associated labeled counter to count the errors.

For instance, [here is the unlock count](https://github.com/mozilla/application-services/blob/741dccfca81afe66c475a57ad643c2e41b43c9ed/components/logins/android/metrics.yaml#L28), and here is the [associated error count](https://github.com/mozilla/application-services/blob/741dccfca81afe66c475a57ad643c2e41b43c9ed/components/logins/android/metrics.yaml#L47) as a labeled counter.

I'm not certain why the collection was designed this way, but it would be much more Glean-like to have all of these recorded under a single labeled counter.  The error labels stay the same with the addition of an "OK" label to record the counts of non-errors that are currently being counted separately.

This creates an unfortunate situation of having to rename the metrics so as to not reuse the existing metrics and just changing the metric type, as well as deprecating the existing metrics and any unification of the data for historical purposes.

tl;dr: Recommend refactoring counter metrics following the pattern of "Counter/Labeled Error Counter" to just "Labeled Counter" recording both "OK" and "Error" counts.
